source/material_model/steinberger.cc

```diff
@@ -268,11 +276,16 @@ namespace aspect
 
       // Evaluate the equation of state properties over all evaluation points
       equation_of_state.evaluate(eos_in, eos_outputs);
+      // std::vector<Vector<double>> viscosities(in.n_evaluation_points(), this->n_compositional_fields());
```

Comment:
Can you remove this and additional commented lines below if they are no longer needed?